### PR TITLE
Ruby Checkpoint Part 1, reciting and testing "The House that Jack Built”

### LINKED ISSUE #1: Write a test that your class returns the following string
### User story
As a developer I want to write a test where the class returns the following string:
```
This is the house that Jack built.
This is the malt that lay in the house that Jack built.
This is the rat that ate the malt that lay in the house that Jack built.
This is the cat that killed the rat that ate the malt that lay in the house that Jack built.
This is the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.
This is the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.
This is the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.
This is the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.
This is the priest all shaven and shorn that married the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.
This is the rooster that crowed in the morn that woke the priest all shaven and shorn that married the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.
This is the farmer sowing his corn that kept the rooster that crowed in the morn that woke the priest all shaven and shorn that married the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.
This is the horse and the hound and the horn that belonged to the farmer sowing his corn that kept the rooster that crowed in the morn that woke the priest all shaven and shorn that married the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.
```

### Acceptance criteria
- [x] Write a test that returns the above string
- [x] Submit a PR with a test

### LINKED ISSUE #2: Write a class that makes the test pass
### User story
As a developer, I want to write a class that makes the test from #1 pass.

### Acceptance criteria

- [x] It has a class that makes the test from #1 pass

### Implementation notes, if any
Blocked by #1

### PR DESCRIPTION
**Pull request by vh8154**
Closes #1 
Closes #2

### CODE REVIEW

**Comment by vh8154** [comment]
This PR is related to Ruby Checkpoint project part 1.

File: .byebug_history [hunk 1]
```diff
@@ -0,0 +1 @@
+c
```
Comment (christinach) [inline .byebug_history]: Can you add this file in .gitignore so that git ignores it?

File: spec/house_spec.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+# require 'byebug'
+
+describe House do
+    it 'returns the house that jack built rhyme' do
+        house = House.new('house.txt')
+        rhyme = house.rhyme
+
+        test_file = File.read('test_text.txt')
+
+        # puts "Test File:\n" + test_file
```
Comment (christinach) [inline spec/house_spec.rb]: Can you remove the comments `# puts ` unless you find them helpful for this project. You can always remove them in the end if you want to keep them until you finish the project.

**Review by christinach — CHANGES_REQUESTED** [review]
Great work @vh8154 ! can you please update the title of the PR and address a couple of comments I added?

**Review by christinach — APPROVED** [review]
Thanks @vh8154 !

**Pull request merged**